### PR TITLE
Spelling Updates Batch 5

### PR DESCRIPTION
**Pull request by kingthorin**
Cherry picked from: https://github.com/zaproxy/zap-extensions/pull/3083 and rebased current.

### CODE REVIEW

**Comment by kingthorin** [comment]
Thanks for correcting that, c & p error 😀

File: addOns/saml/src/main/java/org/zaproxy/zap/extension/saml/SAMLExtension.java [hunk 1]
```diff
@@ -26,7 +26,7 @@
 import org.parosproxy.paros.extension.ExtensionHook;
```
Comment (thc202) [inline addOns/saml/src/main/java/org/zaproxy/zap/extension/saml/SAMLExtension.java:26]: Changelog update.

File: addOns/scripts/src/main/java/org/zaproxy/zap/extension/scripts/PopupEnableDisableScript.java [hunk 2]
```diff
@@ -82,7 +82,7 @@ public boolean isEnableForComponent(Component invoker) {
                     }
```
Comment (thc202) [inline addOns/scripts/src/main/java/org/zaproxy/zap/extension/scripts/PopupEnableDisableScript.java:82]: Same.

File: addOns/tips/src/main/java/org/zaproxy/zap/extension/tips/ExtensionTipsAndTricks.java [hunk 3]
```diff
@@ -126,9 +126,9 @@ public String getDescription() {
     /**
```
Comment (thc202) [inline addOns/tips/src/main/java/org/zaproxy/zap/extension/tips/ExtensionTipsAndTricks.java:126]: Same.

File: addOns/zest/src/main/java/org/zaproxy/zap/extension/zest/ExtensionZest.java [hunk 4]
```diff
@@ -465,7 +465,7 @@ public void redact(ScriptNode node, String replace, String replaceWith, boolean
         this.refreshMessage();
     }
 
-    public void perameterize(
+    public void parametrize(
```
Comment (thc202) [inline addOns/zest/src/main/java/org/zaproxy/zap/extension/zest/ExtensionZest.java:468]: Why not just fix the typo? No need to introduce other variant (e.g. `ZestParameterizeDialog`).

Comment (kingthorin) [reply addOns/zest/src/main/java/org/zaproxy/zap/extension/zest/ExtensionZest.java:468]: I'll have a look.

Comment (kingthorin) [reply addOns/zest/src/main/java/org/zaproxy/zap/extension/zest/ExtensionZest.java:468]: Took me a minute to clue into the exact issue here, but I agree it should have an e after the t.

**Comment by kingthorin** [comment]
Reverted jbrf changes.

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you both!

**Review by psiinon — APPROVED** [review]

**Pull request merged**